internal/internal.go

```diff
@@ -238,6 +238,11 @@ var (
 	// SetBufferPoolingThresholdForTesting updates the buffer pooling threshold, for
 	// testing purposes.
 	SetBufferPoolingThresholdForTesting any // func(int)
+
+	// HTTPSProxyFromEnvironmentForTesting returns the URL of the proxy to use
+	// for testing purposes. It is used to override the `http.ProxyFromEnvironment`
+	// function for testing purposes.
+	HTTPSProxyFromEnvironmentForTesting any // func(*http.Request) (*url.URL, error)
```

Comment:
You can use the correct type here instead of declaring it as `any`. I believe some functions are declared as `any` to avoid circular dependencies among gRPC packages. Since this function doesn't use gRPC types, it should be fine to use the specific type.